services/management/mongodb.go

```diff
@@ -93,6 +93,7 @@ func (s *MongoDBService) Add(ctx context.Context, req *managementpb.AddMongoDBRe
 			MongoDBOptions:    mongoDBOptions,
 			PushMetrics:       isPushMode(req.MetricsMode),
 			DisableCollectors: req.DisableCollectors,
+			LogLevel:          req.LogLevel.String(),
```

Comment:
So we expect that log level always specified? No default value?

Reply:
Default is warn like in issue https://jira.percona.com/browse/PMM-7326
```
while the default should be `–log–level=WARN`
```

So "req.LogLevel.String()" return "warn" if not set in pmm-admin